### ISSUE TITLE
Bundling hangs when content file in browser.json is not found in project

### ISSUE THREAD

**Issue by PeteCopeland** [lasso-js/lasso#184]
When I add a content file for i18n via `browser.json` that doesn't exist in the project, I get the following when I try to bundle directly via the cli:
`Optimizing page "browser"...`

This stays there until `Control + C` is input. When I am trying to hit a page, the `lasso-head` and `lasso-body` tags timeout. 

I'm not sure if this happens with other types of files, or just content files.

**Comment by patrick-steele-idem** [comment]
I don't believe this is a problem with lasso, but rather a problem with the plugin, but I do think that lasso should have a per-dependency timeout that fires before the total timeout so that the error message includes more information. I'm going to retitle this issue.
